DataModeler.cs

```diff
@@ -1,32 +1,88 @@
 ﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json.Linq;
 
 namespace Project1_Group_17
 {
     public class DataModeler
     {
-        public delegate void ParseHandler(string fileName);
-        public void ParseXML(string fileName)
-        {
+        public delegate Dictionary<string, CityInfo> ParseHandler(string fileName);
+        public enum SupportedFileTypes { JSON, XML, CSV };
 
+        public Dictionary<string, CityInfo> ParseXML(string fileName)
```

Comment:
He specified that they all return void. Later he says they return a dictionary, but I assume he means modifies an existing, class scope, dictionary

Reply:
fixed